patientsummary/triggers.go

```diff
@@ -0,0 +1,40 @@
+package patientsummary
+
+var (
+	triggers = map[string]struct{}{
+		"LEGACY_DATA_ADDED":       {},
```

Comment:
Somewhat unfortunate that these cant be pulled in from somewhere dynamic, summary client is already being pulled into clinic-worker, I will take note of this and define these in the summary client in the future.